public/js/actions/pause.js

```diff
@@ -135,11 +135,15 @@ function loadObjectProperties(grip) {
  */
 function addExpression(expression) {
   return ({ dispatch, getState }) => {
+    const lastExpression = getExpressions(getState()).toJS().pop();
+    const id = parseInt(expression.id, 10) ||
+      (lastExpression ? lastExpression.id : -1) + 1;
```

Comment:
One approach would be to have a variable at the top of the module `let expressionId = 1` and then in here do `{ id: expressionId++ }`